.github/workflows/chart.yaml

```diff
@@ -0,0 +1,26 @@
+name: publish_helm_chart
+
+on:
+  push:
+    branches:
+      - main
+      - master
+    paths:
+      - ".github/workflows/chart.yaml"
+      - "charts/**"    
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+      - name: Publish Helm chart
+        uses: stefanprodan/helm-gh-pages@v1.4.1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
a comment explaining this may be helpful - i'm not too familiar with github action security

Reply:
Added a comment with reference to the doc